config/locales/de.yml

```
@@ -84,16 +84,14 @@ de:
       byline: eine Dialog-Recherche von <a class="Link" href="https://tactile.news" target="_blank">tactile.news</a>
     onboarding_signal_link:
       heading: Fast geschafft.
-      text: Damit wir Ihnen zukünftig auf Signal schreiben können, müssen Sie uns zuerst kontaktieren.
-      action: Senden Sie uns hierzu eine Nachricht auf Signal. Ein kurzes „Hallo” reicht aus! Unsere Nummer ist <strong>%{number}</strong>.
-      link: Zur Signal-App
-      copy: Handynummer kopieren
-      help:  Über den Button gelangen Sie direkt zu unserem Kontakt in der Signal-App. Alternativ können Sie unsere Nummer kopieren.
+      text: Verbinden Sie Ihr Signal-Konto in der Signal-App, um die Anmeldung abzuschließen.
+      steps:
+        - "Finden Sie uns auf Signal. Suchen Sie dazu nach <strong>%{signal_server_phone_number}</strong>."
+        - "Wenn Sie unser Signal-Konto gefunden haben, geben Sie diesen Code ein, sobald sie danach gefragt werden: <strong>%{token}</strong>"
```

```suggestion
        - "Wenn Sie unser Signal-Konto gefunden haben, geben Sie diesen Code ein, sobald Sie danach gefragt werden: <strong>%{token}</strong>"
```

Very formal German..

---

Yes, I remember that we wanted informal. I copy/pasted this from the `onboarding_telegram_fallback` component's steps... Could you suggest the informal version?